update wenxin llm

Weaxs opened the pull request:
# Description

update wenxin llm

- [ERNIE-4.0-8K](https://cloud.baidu.com/doc/WENXINWORKSHOP/s/clntwmv7t) (old name: ERNIE-Bot-4.0)
- [ERNIE-3.5](https://cloud.baidu.com/doc/WENXINWORKSHOP/s/jlil56u11)
    - ERNIE-3.5-8K (old name: ERNIE-Bot-8K)
    - ERNIE-3.5-8K-0205
    - ERNIE-3.5-8K-1222
    - ERNIE-3.5-4K-0205 (old name: ERNIE-Bot)
- [ERNIE-Speed](https://cloud.baidu.com/doc/WENXINWORKSHOP/s/klqx7b1xf)
  -   ERNIE-Speed-8K
  - ERNIE-Speed-128K
- [ERNIE-Lite](https://cloud.baidu.com/doc/WENXINWORKSHOP/s/4lilb2lpf)
   - ERNIE-Lite-8K-0922 (old name: ERNIE-Bot-Turbo)
   - ERNIE-Lite-8K-0308


Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement，including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

Weaxs committed:
update wenxin ernie

Weaxs committed:
update ernie-lite-8k-0922

bowenliang123 commented:
One major question first, is that `ernie` model a Baidu's product `文心一言`?
Why it's described as `wechat` in the title and description ?

Weaxs commented:
> One major question first, is that `ernie` model a Baidu's product `文心一言`? Why it's described as `wechat` in the title and description ?

yeah, `文心大模型3.5` is `ernie-3.5-8k` and `文心大模型4.0` is `ernie-4.0-8k` in fact

Yeuoly commented:
Thanks a lot!
But I have some advice here, for forward compatibility, A deprecated field should be set in the yaml file instead of removing it,  you can refer to `core.model_runtime.model_providers.openai.llm.text-davinci-003.yaml`, `deprecated: true` will be found there.

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-4.0-8k.yaml:
```diff
@@ -1,12 +1,12 @@
-model: ernie-bot
+model: ernie-4.0-8k
```
a model name should not be changed once it has been used for a period of time, otherwise it will lead the configuration of some apps to be invalid.

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-4.0-8k.yaml:
```diff
@@ -15,16 +15,17 @@ parameter_rules:
-  - name: presence_penalty
-    use_template: presence_penalty
+  - name: penalty_score
+    use_template: penalty_score
```
there is not a template named penalty_score in our default templates, you can refer to `core.model_runtime.entities.defaults`

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml:
```diff
@@ -0,0 +1,27 @@
+    max: 1.0
+    default: 0.7
+  - name: max_output_tokens
+    use_template: max_output_tokens
```
but also, max_output_tokens is not our template, you should alias `max_tokens` to `max_output_tokens` in llm.py

Weaxs committed:
fix yaml

Weaxs committed:
add api_bases for deprecated

crazywoola reviewed (commented):
Please resolve the lint errors in CI build.

Weaxs committed:
add blank line at the end of file

Weaxs committed:
ernie bot
max_tokens -> max_output_tokens
presence_penalty -> penalty_score

default use ernie-3.5-8k in test case

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-speed-8k.yaml:
```diff
@@ -0,0 +1,30 @@
+    use_template: max_tokens
+    default: 1024
+    min: 1
+    max: 8000
```
max_output_tokens of `ernie-speed-8k` is limited to 1024, refer to https://cloud.baidu.com/doc/WENXINWORKSHOP/s/klqx7b1xf

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-speed-128k.yaml:
```diff
@@ -0,0 +1,30 @@
+    use_template: max_tokens
+    default: 1024
+    min: 1
+    max: 128000
```
It's also 1024, and the minimum is 2

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml:
```diff
@@ -0,0 +1,30 @@
+  - name: max_tokens
+    use_template: max_tokens
+    default: 1024
+    min: 1
```
2 and 1024

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-speed-appbuilder.yaml:
```diff
@@ -0,0 +1,30 @@
+    min: 0
+    max: 1.0
+    default: 0.7
+  - name: max_tokens
```
this parameter seems dose not exist

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0922.yaml:
```diff
@@ -0,0 +1,30 @@
+  - name: max_tokens
+    use_template: max_tokens
+    default: 1024
+    min: 1
```
2 and 1024

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-4.0-8k.yaml:
```diff
@@ -0,0 +1,37 @@
+  - name: max_tokens
+    use_template: max_tokens
+    default: 256
+    min: 1
```
2 and 2048

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-3.5-8k.yaml:
```diff
@@ -0,0 +1,37 @@
+    use_template: top_p
+  - name: max_tokens
+    use_template: max_tokens
+    default: 1024
```
2 and 2048

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-3.5-8k-1222.yaml:
```diff
@@ -0,0 +1,37 @@
+  - name: max_tokens
+    use_template: max_tokens
+    default: 1024
+    min: 1
```
2 and 2048

Yeuoly commented on api/core/model_runtime/model_providers/wenxin/llm/ernie-3.5-8k-0205.yaml:
```diff
@@ -0,0 +1,37 @@
+  - name: max_tokens
+    use_template: max_tokens
+    default: 1024
+    min: 1
```
2 and 2048

Yeuoly reviewed (changes requested):
Thanks a lots, bu after read the document of ernie not, there is something small details need to be revised

Weaxs committed:
fix max_tokens scopes

Yeuoly reviewed (approved):
LGTM
